src/VDPDataStore.h

```diff
@@ -22,6 +22,10 @@ class VDPDataStore : public QObject, public SimpleHexRequestUser
 
 	size_t getVRAMSize() const;
 
+	bool getRegisterLatchAvailable() const;
+	bool getPaletteLatchAvailable() const;
+	bool getDataLatchAvailable() const;
```

Comment:
Personal preference: such very simple getters could be implemented directly in the .h file:
```
bool getRegisterLatchAvailable() const { return registerLatchAvailable; }
bool getPaletteLatchAvailable() const { return paletteLatchAvailable; }
bool getDataLatchAvailable() const { return dataLatchAvailable; }
```

BTW: whenever I say "personal preference" feel free to disagree and keep your own personal preference.